pkg/storageops/gce/gce.go

```diff
@@ -187,8 +187,30 @@ func (s *gceOps) Create(
 }
 
 func (s *gceOps) Delete(id string) error {
-	_, err := s.service.Disks.Delete(s.inst.Project, s.inst.Zone, id).Do()
-	return err
+	ctx := context.Background()
+	found := false
+	req := s.service.Disks.AggregatedList(s.inst.Project)
+	if err := req.Pages(ctx, func(page *compute.DiskAggregatedList) error {
+		for _, diskScopedList := range page.Items {
+			for _, disk := range diskScopedList.Disks {
+				if disk.Name == id {
+					found = true
+					_, err := s.service.Disks.Delete(s.inst.Project, path.Base(disk.Zone), id).Do()
+					return err
+				}
+			}
+		}
+		return nil
+	}); err != nil {
+		logrus.Errorf("failed to list disks: %v", err)
+		return err
+	}
+
+	if !found {
+		return fmt.Errorf("disk: %s as it wasn't found", id)
```

Comment:
nit: Failed to delete disk: %s as it wasn't found.

Reply:
done